<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TRPL-PUN-0115" failures="0" tests="22" name="criticalTestScripts.Reviewer" time="3249.284" errors="1" timestamp="2024-09-28T02:11:32 IST" skipped="0">
  <testcase classname="criticalTestScripts.Reviewer" name="CompletedStatutoryCri" time="99.563"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="CompletedInternalCri" time="139.435"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="OverdueStatutoryCri" time="103.879"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="OverdueInternalCri" time="106.031"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="ReviewCountStatutoryApprove" time="232.989"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="ReviewCountStatutoryReject" time="220.867"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="PendingReviewInternalCri1" time="223.157"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="ReviewCountInternalReject" time="220.262"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="DueButNotSubmitted" time="101.020"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="DueButNotSubmittedIn" time="95.523"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="PendingForReview" time="145.942"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="PendingForReviewIn" time="164.436"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="Rejectedtask" time="88.868"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="PendingForReviewTaskRejected" time="149.835"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="RejectedtaskIn" time="97.376"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="PendingForReviewTaskInRejected" time="150.787"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="Compliancecalendar" time="145.542"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="DetailedReport1" time="133.003"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="DetailedReportIn" time="132.161"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="AssignmentReport" time="87.104">
    <error message="Cannot read the array length because &quot;dirContents&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot read the array length because "dirContents" is null
at cfo.CFOcountPOM.Assignment(CFOcountPOM.java:8678)
at cfo.CFOcountPOM.AssignmentReport(CFOcountPOM.java:8528)
at criticalTestScripts.Reviewer.AssignmentReport(Reviewer.java:654)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- AssignmentReport -->
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="ComplianceDocumentsSat" time="183.453"/>
  <system-out/>
  <testcase classname="criticalTestScripts.Reviewer" name="ComplianceDocumentsInter" time="228.051"/>
  <system-out/>
</testsuite> <!-- criticalTestScripts.Reviewer -->
